config/routes.rb

```diff
@@ -76,6 +76,7 @@
 
   get "/ask-questions", to: "pages#ask_questions"
   get "/helpdesk-request-submitted", to: "pages#helpdesk_request_submitted"
+  get "/delayed-zendesk-ticket", to: "pages#delayed_zendesk_ticket"
```

Comment:
I don't think we should make `zendesk-ticket` a part of our URLs, because it's an implementation detail as far as the user is concerned. Instead, we could try:

```suggestion
  get "/helpdesk-request-delayed", to: "pages#helpdesk_request_delayed"
```